eap-xp/runtime-image/image.yaml

```diff
@@ -0,0 +1,74 @@
+schema_version: 1
+
+name: "jboss-eap-7/eap-xp1-openjdk11-runtime-openshift-rhel8"
+description: "The JBoss EAP XP 1.0 openjdk11 runtime image"
+version: "1.0"
+from: "ubi8:8-released"
+labels:
+    - name: "com.redhat.component"
+      value: "jboss-eap-7-xp1-openjdk11-runtime-openshift-rhel8-container"
+    - name: "io.k8s.description"
+      value: "Base runtime image to run EAP XP server and application"
+    - name: "io.k8s.display-name"
+      value: "JBoss EAP XP runtime image"
```

Comment:
we don't have the version here

Reply:
Hmm, yeah so we can either put the version in both or leave it out of both. It probably makes sense to leave it out in both, as the version should come from the image's tagged version.

Reply:
I dropped the version, so it shouldn't be there in either case.